cmd/acra-translator/common/service.go

```diff
@@ -81,7 +83,7 @@ func (service *TranslatorService) Decrypt(ctx context.Context, acraStruct, clien
 
 	data, decryptErr := service.handler.DecryptWithHandler(handler, acraStruct, dataContext)
 	if decryptErr != nil {
-		base.AcrastructDecryptionCounter.WithLabelValues(base.DecryptionTypeFail).Inc()
+		base.AcrastructDecryptionCounter.WithLabelValues(base.LabelStatusFail).Inc()
```

Comment:
can you add for every legacy metric changes TODO with a reminder to remove it after 1-2 version due to migrated metrics to new? and will be great to create task for that for the future )